subprojects/core-api/build.gradle.kts

```diff
@@ -59,7 +59,7 @@ strictCompile {
     ignoreRawTypes() // raw types used in public API
 }
 
-integTest.usesJavadocCodeSnippets = true
+integTest.generateDefaultAutoTestedSamplesTest = false
```

Comment:
❓ Why?

If we need this, add a comment explaining why.

Reply:
Done!